config/1.0.0/sample_config_1.0.0.json

```diff
@@ -18,96 +18,151 @@
     "SPL[1234].+": "SMAP",
     "VIIRS_NPP-.+-L2P": "VIIRS_PO"
   },
-  "Excluded_Science_Variables": [
+  "ExcludedScienceVariables": [
     {
       "Applicability": {
         "Mission": "ICESat2"
       },
-      "Variable_Pattern": [
+      "VariablePattern": [
         "/quality_assessment/.*",
         "/orbit_info/.*",
         "/atlas_impulse_response/.*"
       ]
     }
   ],
-  "Required_Fields": [
+  "RequiredVariables": [
     {
       "Applicability": {
         "Mission": "GEDI"
       },
-      "Variable_Pattern": [
+      "VariablePattern": [
         ".*shot_number"
       ]
     }
   ],
-  "ProductEpochs": [
+  "CFOverrides": [
     {
       "Applicability": {
-        "Mission": "ICESat2"
+        "Mission": "SMAP",
+        "ShortNamePath": "SPL4.*"
       },
-      "Epoch": "2005-01-01T00:00:00.000000"
+      "Attributes": [
+        {
+          "Name": "grid_mapping",
+          "Value": "/EASE2_global_projection"
+        }
+      ]
     },
     {
       "Applicability": {
-        "Mission": "GEDI"
+        "Mission": "SMAP",
+        "ShortNamePath": "SPL4.*",
+        "VariablePattern": "/EASE2_global_projection"
       },
-      "Epoch": "2018-01-01T00:00:00.000000"
-    }
-  ],
-  "Grid_Mapping_Data": [
-    {
-      "Grid_Mapping_Dataset_Name": "EASE2_Global",
-      "grid_mapping_name": "lambert_cylindrical_equal_area",
-      "standard_parallel": 30.0,
-      "longitude_of_central_meridian": 0.0,
-      "false_easting": 0.0,
-      "false_northing": 0.0
```

Comment:
So this is all just deleted, and not added because it wasn't used?

Reply:
The `ProductEpoch` was entirely unused, so I just deleted it. I moved the content from `Grid_Mapping_Data` to the `CFOverrides`/`MetadataOverrides` section, though, just to provide examples for how that information would be included in the future. (In theory that sample configuration file should probably be slimmed down to a minimum number of examples, but I think that some of those rules were placed there so DAS has them somewhere)